include/deal.II/grid/grid_generator.h

```diff
@@ -1130,7 +1130,7 @@ namespace GridGenerator
    * assign appropriate boundary objects through Triangulation::set_boundary()
```

Comment:
We should not mention `set_boundary` any more.

Reply:
I assume we have to fix that other places as well.

Reply:
true; perhaps we can leave this for another PR.